prov/util/src/util_coll.c

```diff
@@ -673,37 +674,29 @@ int ofi_join_collective(struct fid_ep *ep, fi_addr_t coll_addr,
 		coll_mc = (struct util_coll_mc *) coll_addr;
 	}
 
-	ret = util_coll_mc_alloc(&new_coll_mc);
-	if (ret)
-		return ret;
-
 	if (util_coll_cid_initialized == 0) {
 		util_coll_init_cid(util_coll_cid);
 		util_coll_cid_initialized = 1;
 	}
 
-	new_coll_mc->mc_fid.fid.fclass = FI_CLASS_MC;
-	new_coll_mc->mc_fid.fid.context = context;
-	new_coll_mc->mc_fid.fid.ops = &util_coll_fi_ops;
-	new_coll_mc->mc_fid.fi_addr = (uintptr_t) new_coll_mc;
-
-	new_coll_mc->ep = ep;
```

Comment:
The changes surrounding this area are unrelated to writing an entry to the CQ.  Changes fall into wrong patch?